assets/messages.yml

```diff
@@ -351,6 +351,20 @@ messages:
         This is known to be an issue involving detection of the Java runtime.
         Please take a look at the Moderator’s Note on that ticket to see solutions to your issue.
 
+        %quick_links%
+      fillname: []
+  duplicate-of-mc-108:
+    - project: mc
+      name: Duplicate of MC-108
+      shortcut: quasi
+      message: |-
+        *Thank you for your report!*
+        We're tracking this issue as *MC-108*, so this ticket is being resolved and linked as a *duplicate*.
+
+        That ticket has already been resolved as working as intended, which means this is not considered a bug and won't be fixed. Please do not leave a comment on the linked ticket.
+
+        If you would like to learn more about this particular feature, see [this article|https://minecraft.gamepedia.com/Tutorials/Quasi-connectivity] on the Minecraft Wiki.
```

Comment:
I would substitute "behavior" for "feature" here.
For people who don't expect this, calling it a feature is a bit far fetched, and it's more of a bug that turned out useful and was not removed.

Also I get tech-community-vibes from this. Certain members of that community like to complain about certain """""features""""" being removed.

Reply:
Would you mind creating a pull request for it?